GVFS/GVFS.NativeHooks.Common/common.posix.cpp

```diff
@@ -31,22 +31,22 @@ PATH_STRING GetGVFSPipeName(const char *appName)
     }
     
     PATH_STRING finalRootPath(GetFinalPathName(enlistmentRoot));
-    size_t resultLength = strlcpy(enlistmentRoot, finalRootPath.c_str(), sizeof(enlistmentRoot));
-    if (resultLength >= sizeof(enlistmentRoot))
+    size_t enlistmentRootLength = strlen(finalRootPath.c_str());
+    if (enlistmentRootLength + 2 > sizeof(enlistmentRoot))
```

Comment:
Could you add a comment explaining why we're adding 2?

I'm assuming it's because we might have to add a `/` below.

Reply:
Yes, exactly.